code/modules/mining/mine_turfs.dm

```diff
@@ -18,7 +18,7 @@
 	var/last_act = 0
 	var/datum/geosample/geologic_data
 	var/excavation_level = 0
-	var/list/finds = list()//no longer null to prevent those pesky runtime errors
```

Comment:
I can just imagine an oldcoder going "eh well we COULD use sanity but why not just create a new list for every single mineral turf that won't cause problems right?"